test/exercism/user_test.rb

```diff
@@ -18,8 +18,13 @@ def test_flipper_id_is_username
     assert_equal 'spocktocat', user.flipper_id
   end
 
+  def build_github_user(id, username, email, avatar_url)
```

Comment:
The reason for that, since I was asked locally, and someone else may have the same question... we don't really care if that `GitHubUser` is bred, built, manufactured, instantiated, genetically cloned, we only care about the return.  The verb hints at how what is returned is created.